### PR TITLE
Canonize service port env vars so K8S doesn't break them with tcp://<…

### PR DESCRIPTION
**Pull request by andytson-inviqa**
…ip>: prepended

Also reduce _HOST_PORT usage keeping it standard _PORT

### CODE REVIEW

File: ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh [hunk 1]
```diff
@@ -253,6 +266,16 @@ function test_remote_ports() {
   done
 }
 
+function deprecate_env_var() {
```
Comment (sroze) [inline ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh]: ❤️

File: magento1/etc/confd/templates/magento/local.xml.tmpl [hunk 2]
```diff
@@ -37,13 +37,13 @@
             </default_setup>
         </resources>
         <session_save><![CDATA[memcached]]></session_save>
-        <session_save_path><![CDATA[{{ getenv "MEMCACHED_HOST" }}:{{ getenv "MEMCACHED_HOST_PORT" }}]]></session_save_path><!-- e.g. for memcache session save handler tcp://10.0.0.1:11211?persistent=1&weight=2&timeout=10&retry_interval=10 -->
+        <session_save_path><![CDATA[{{ getenv "MEMCACHED_HOST" }}:{{ getenv "MEMCACHED_PORT" }}]]></session_save_path><!-- e.g. for memcache session save handler tcp://10.0.0.1:11211?persistent=1&weight=2&timeout=10&retry_interval=10 -->
```
Comment (sroze) [inline magento1/etc/confd/templates/magento/local.xml.tmpl]: Could you add `MEMCACHED_PORT`  and `MEMCACHED_HOST` to the M1 README?

**Comment by fratcliffe** [comment]
Tested drupal, magento1, magento2 and spryker builds locally.

File: magento1/README.md [hunk 3]
```diff
@@ -95,12 +95,14 @@ MAGENTO_MODE | Used to set Magento mode. If set to "production", static contents
 MAGENTO_RUN_CODE_MAPPING | Mapped to http_host and default store name. First part of the value is the host name and second part is magento's store code (separated by space). Don't forget to add ";" at the end. | - | magento_web.docker default;
 FRONTEND_COMPILE_LANGUAGES | Used during static content deployment. It can be multiple language codes. | language code(s) separated by space | en_GB
 MAGENTO_DEPENDENCY_INJECTION_COMPILE_COMMAND | Magento DI compile command | - | bin/magento setup:di:compile
-MAGENTO_CRYPT_KEY | Magneto crypt key | - | -
```
Comment (sroze) [inline magento1/README.md]: I liked Magneto 😂

**Pull request merged**